wger/exercises/views/muscles.py

```diff
@@ -61,7 +65,7 @@ def get_context_data(self, **kwargs):
 
 
 class MuscleAdminListView(LoginRequiredMixin,
-                          PermissionRequiredMixin, MuscleListView):
+                          PermissionRequiredMixin, ListView):
```

Comment:
Why did you switch from MuscleListView which seems to have been defined by the original developers to ListView?

Reply:
I overrode the `get_queryset` so that's why I added ListView to allow return the list of all muscles